src/MainCase/MainCase.scad

```diff
@@ -266,8 +266,8 @@ module MainCase(without_inserts=false) {
             polygon(polyRound([
               [0, 0, 0],
               [UsbCharger_width, 0, 0],
-              [UsbCharger_width, UsbCharger_depth+wall_thickness/2, 3],
-              [0, UsbCharger_depth+wall_thickness/2, 0],
+              [UsbCharger_width, UsbCharger_depth+2*extrude_width, 3],
+              [0, UsbCharger_depth+2*extrude_width, 0],
```

Comment:
I don't know -- this is dangerous, if somebody has a 0.8 nozzle (@tobst does this, I think) then this will be 1.6mm, which is way too much I fear. We should try to pick the smallest multiple of `extude_width` that is `>= 0.6` or so. This might work:

```openscad
ceil(0.6 / extrude_width) * extrude_width
```

Reply:
it doesn't have to be a multiple necessarily, but with .4 mm nozzles a single-extrusion wall is dangerous.

we can also just leave it at .92 mm fixed - any extrusion width > .46 will then produce a single wall extrusion.